crypto/store/store_init.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2001-2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/err.h>
+#include "internal/store.h"
+#include "store_local.h"
+
+static CRYPTO_ONCE store_init = CRYPTO_ONCE_STATIC_INIT;
+DEFINE_RUN_ONCE_STATIC(do_store_init)
+{
+    return OPENSSL_init_crypto(0, NULL)
```

Comment:
I do remember having a Wtf moment with something almost but not exactly like that. I'll try and see if I can reproduce it. 

It's also possible that conditions have changed since I wrote this code, and that the issue is therefore not reproducible any more. 

Ah well, if you tell me this is absolutely needless, I can remove it. If it turns out to be a bug, we add it back...